transport/src/main/java/io/scalecube/transport/MessageHandler.java

```diff
@@ -19,9 +19,9 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MessageHandler.class);
 
-  private final Subject<Message, Message> incomingMessagesSubject;
+  private final Processor<Message,Message> incomingMessagesSubject;
```

Comment:
FluxProcessor instead of Processor and FluxSink for sending data manually (if it supposed that this method will be handled by the several threads, consider additional FluxProcessor.serialized()